### PR TITLE
[Feature] Read and display summary 

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description
Initial commit to read data from Google spreadsheet summary view and respond to user with custom Slack summary message

## Implementation
- Adds initial summary end point
- Adds new response structure for summary test

## Screenshot
Screenshot 2020-02-17 at 13 08 43

## .env changes
n/a

## Additional requirements
n/a

Sums it up

### CODE REVIEW

**Comment by railsagainstignorance** [comment]
"Summary count of all missions"
The count of what, tho? Products? Projects? Completed? Inflight?
Is this the wording from a cell in the underlying spreadsheet? In which case, it is easy to reword it to provide more context, so no need to change the PR.

**Comment by Sqyphen** [comment]
Summary content is taken from the spreadsheet only, so can be edited.
The summaries currently in the sheet are examples, sheets with calculations in, to show that the data can be pulled through.
They can be updated with summaries of interest later.
